components/status/StatusCard.vue

```diff
@@ -79,7 +79,8 @@ const showReplyTo = $computed(() => !replyToMain && !directReply)
   <div
     :id="`status-${status.id}`"
     ref="el"
-    relative flex="~ col gap1" p="l-3 r-4 b-2"
+    relative flex="~ col gap1"
+    p="l-3 r-4 b-2" rtl-p="l-4 r-3"
```

Comment:
this should also be fixed

Reply:
Thanks so much, please forgive my mistake😅